duckduckgo/macos-browser
# Duck Player Page Cleanup

Task/Issue URL: https://app.asana.com/0/970019368605465/1203311633129277
Tech Design URL:
CC:

**Description**:
Addresses PR feedback, removes unused code and simplifies and the Duck Player Page.



**Steps to test this PR**:
- Regression-test the entire functioanlity of the Duck Player page, there should be no regressions. These things should work as expected:
   - Videos should load (including with timestamp)
   - Logo and tooltip on the (i)-icon
   - Inactivity timer to hide / show the toolbar beneath the video
   - "Always open"-setting 
   - Setting & YouTube buttons


**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
